server/routes/charts.js

```diff
@@ -13,6 +13,7 @@ import editChartPage from '../templates/EditChart.template'
 import { legend } from '../helpers/chartsHelpers'
 import { graphDataController } from '../controllers/chartsController'
 import { routes } from '../utils/routes'
+import { targetValuesFields } from '../../views/fe-utils/targetValuesUtil'
```

Comment:
I think last time we imported a front end module to the server directory the app crashed.

Reply:
Thank you for the heads up. I moved it to a root directory, https://github.com/AMP-SCZ/dpdash/pull/174/commits/8774bda0778065446b3510aa70c58b59b50d93f1

Reply:
I haven't proven this but my hunch is that the front and back ends can only import from their directories (or at least one of them can only import from its directory). Do you know how we could test that this will work? It would be great to not duplicate logic but it may not be avoidable.

Reply:
Thanks for that @mikestone14, I think going the less risky route of recreating the logic. 
https://github.com/AMP-SCZ/dpdash/pull/174/commits/57a87a23977d766c8410cbdcdb1d034080084057